scripts/exdump_monitor.sh

```diff
@@ -770,7 +770,7 @@ elif [ $cycp -eq 23 ]; then   # (22.50 - 23.49 Z)
    unset SKIP_006063 # reflectivity 23Z
 fi
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 nexrad
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 nexrad
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.